src/middlelayer/token_db_handler.rs

```diff
@@ -0,0 +1,24 @@
+use crate::database::crud::CrudDb;
+use crate::database::dsls::user_dsl::User;
+use crate::middlelayer::db_handler::DatabaseHandler;
+use crate::middlelayer::token_request_types::DeleteToken;
+use anyhow::{anyhow, Result};
+use diesel_ulid::DieselUlid;
+
+impl DatabaseHandler {
+    pub async fn create_token(&self) {
+        todo!()
```

Comment:
On a database perspective this is straight, forward, create the token (with uuid) and add it to the jsonb attributes of the user